phpBB/phpbb/event/kernel_terminate_subscriber.php

```diff
@@ -35,7 +35,7 @@ public function on_kernel_terminate(PostResponseEvent $event)
 	public static function getSubscribedEvents()
 	{
 		return array(
-			KernelEvents::TERMINATE		=> 'on_kernel_terminate',
+			KernelEvents::TERMINATE		=> array('on_kernel_terminate', -10000),
```

Comment:
There is no case for not running this last, right? So make this something like `-PHP_INT_MAX` or `~PHP_INT_MAX` ?